appveyor.yml

```diff
@@ -1,14 +1,26 @@
 version: 2.0.{build}
-branches:
-  only:
-  - master
-  - 2.0
 image: Visual Studio 2017
 configuration: Debug
-platform: Any CPU
+assembly_info:
+  patch: true
+  file: '**\AssemblyInfo.*'
+  assembly_version: '{version}'
+  assembly_file_version: '{version}'
+  assembly_informational_version: '{version}'
```

Comment:
Be aware that this does noting with the new csproj format. In this case the assembly info is generated as part of the build which occurs after app veyor attempts to find and patch the AssemblyInfo files.
I see you are handling those cases in the dotnet_csproj below. Is this still needed?